sdk/src/main/java/com/mapbox/maps/MapboxMap.kt

```diff
@@ -1709,8 +1709,7 @@ class MapboxMap :
   @MapboxExperimental
   override fun getMapProjection(): MapProjection {
     checkNativeMap("getMapProjection")
-    val value = nativeMap.mapProjection
-    return MapProjectionUtils.fromValue(value)
+    return MapProjection.Mercator
```

Comment:
ok with mercator here as this will be revisited later.
however we need to make sure that tests don't fail or (if they do) - comment them out for now, I'll handle that after this PR is merged